Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -2964,104 +3111,40 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "123":
-    id: "123"
-    taskid: e02273a0-e7c1-4084-8073-935de559bf4d
+  "133":
+    id: "133"
+    taskid: 2083b835-6639-4b1b-8fc9-8c3e6f13dd0e
     type: regular
     task:
-      id: e02273a0-e7c1-4084-8073-935de559bf4d
+      id: 2083b835-6639-4b1b-8fc9-8c3e6f13dd0e
       version: -1
-      name: GenerateSummaryReport - Analysis
-      description: Generate an ASM Alert Summary report.
-      scriptName: GenerateASMReport
+      name: Set playbook stage grid field (remediation)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Instead of a value, you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```